### ISSUE TITLE
Error handler seems to have Type problems.

### ISSUE THREAD

**Issue by dafstone** [Munter/express-compile-sass#5]
I've recently installed express-compile-sass in a new project.  Looks like a great project.   Works great, except when it throws an error.  (it's otherwise compiling and caching fine).

Any time it encounters an error, rather than reporting the actual error it throws an error on it's own error handler.  See below:

```
Compiling sass file: -app-path-/public/stylesheets/style.sass

-app-path-/node_modules/express-compile-sass/lib/index.js:114
mpile-sass:\n  Syntax error in ' + req.originalUrl + ' line ' + err.replace('s
                                                                    ^
TypeError: Object #<Object> has no method 'replace'
    at sassError (-app-path-/node_modules/express-compile-sass/lib/index.js:114:110)
    at options.error (-app-path-/node_modules/node-sass/lib/index.js:159:7)

```

As far as I can tell this seems to be throwing an issue in handling the type on the error returned from node-sass, and therefore is outside of the scope of the code local to my app.  I think.  I did try this against releases 2.1.0 and 2.0.4 as well and encountered the same error - so I'm surprised.  I'm new to express, but I've spent a not insignificant amount of time looking at this and I can't see that this has another cause.

Thanks!

**Comment by Munter** [comment]
Are you by any chance using node-sass@2.0.0-beta?

There seem to be breaking changes in it. I made an update a few days ago to cater for normal usage, but didn't test error conditions

**Comment by Munter** [comment]
Yup, that seems to be it. The unit tests fail with that version. I'll look into it

**Comment by dafstone** [comment]
Yep - I was and rolled back to 1.1.4 and it's all working again.  Thanks very much!  I'll keep an eye out for the update.   Thanks for what you're doing - your approach to cache handling is exactly what I was looking for.

**Comment by Munter** [comment]
Published in v2.1.2.

If you like this approach to precompiling, cache handling and live reloading then you might be interested in taking a look at https://github.com/One-com/livestyle as well. This is the project I originally intended to integrate with in order to also get live reloading

**Comment by dafstone** [comment]
Thanks very much.  Just updated again and everything is working perfectly.  And thanks for turning me on to livestyle.
